README.md

```diff
@@ -12,11 +12,45 @@ cd $GOPATH/src/sdx-encrypted-submitter
 go get -u github.com/streadway/amqp
 go get -u gopkg.in/yaml.v2
 go build
+go install
+```
+
+### Preparing Keys
+
+This requires that new keys are generated and added to sdx-decrypt so that this utility does not 
+need knowledge of existing eq and sdx keys. To do this we need to generate 2 pairs of keys, one pair that 
+is supplemental to existing eq keys and the other which is supplemental to sdx keys.
+To generate the key pairs we need to create keys with the correct name format .
+```
+sdc-<service>-submission-<key-use>-<key-type>-<version>.pem
+```
+Where 
+<service> is set to 'submitter' for upstream keys and 'sdx' for sdx keys
+Note 'submitter' is used in place of 'eq' so that we do not clash with eq keys.
```

Comment:
This should probably be in italics or something distinctive.  Also the previous sentence doesn't have a full stop.

Reply:
Done